ang_scripts/force_password_change.py

```diff
@@ -0,0 +1,17 @@
+import django
+import logging
+
+logger = logging.getLogger(__name__)
+
+django.setup()
+
+
+def force_password_change():
+    from registry.groups.models import CustomUser
+    logger.info("Forcing password change for all users")
+    count = CustomUser.objects.all().update(force_password_change=True)
```

Comment:
Just as a note you can leave out the `all().`. ie `count = CustomUser.objects.update(force_password_change=True)`